php.ini-development

```diff
@@ -630,16 +630,14 @@ ignore_repeated_source = Off
 
 ; This directive determines which super global arrays are registered when PHP
 ; starts up. G,P,C,E & S are abbreviations for the following respective super
-; globals: GET, POST, COOKIE, ENV and SERVER. There is a performance penalty
-; paid for the registration of these arrays and because ENV is not as commonly
-; used as the others, ENV is not recommended on productions servers. You
-; can still get access to the environment variables through getenv() should you
-; need to.
+; globals: GET, POST, COOKIE, ENV and SERVER.
+; In environments where _ENV is not registered, it is still possible to get
+; access to the environment via getenv()
 ; Default Value: "EGPCS"
-; Development Value: "GPCS"
-; Production Value: "GPCS";
-; https://php.net/variables-order
-variables_order = "GPCS"
+; Development Value: "EGPCS"
+; Production Value: "EGPCS";
```

Comment:
Similarly these lines should be removed.

Reply:
Other close by and related settings declare here their defaults, ini and production values, so it seems fine to follow that pattern.

Reply:
They only do that when any of them differ from the built-in default.